recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,2 @@
 pytorch_version:
-  - 1.8.0
-  - 1.9.0
-  - 1.9.1
```

Comment:
Can you delete this file and pull in the migration file manually?

Reply:
Sure, which one should I pull in? The torchvision feedstock had a lot and I'm not sure which one is needed here.

Reply:
This file is the one you need to copy over

https://github.com/conda-forge/torchvision-feedstock/blob/master/.ci_support/migrations/pytorch110.yaml

Reply:
done!